src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -5,63 +5,115 @@ import moment from 'moment';
 import * as customPropTypes from '../prop-types';
 
 // Helper to derive the download link props.
-const deriveLinkProps = form => {
+const deriveLinkPropsFromFormURL = url => {
   const linkProps = {};
+  if (!url) return linkProps;
 
-  const isSameOrigin = form?.attributes?.url.startsWith(window.location.origin);
-  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+  const isSameOrigin = url.startsWith(window.location.origin);
+  const isPDF = url.toLowerCase().includes('.pdf');
 
   if (!isSameOrigin || !isPDF) {
     // Just open in a new tab if we'd otherwise hit a CORS issue or if the form URL isn't a PDF.
     linkProps.target = '_blank';
   } else {
     // Use HTML5 `download` attribute.
-    linkProps.download = true;
+    linkProps.download = decodeURI(url.split('/').pop());
+    if (isPDF) linkProps.type = 'application/pdf';
   }
 
   return linkProps;
 };
 
+// builds results title
+const buildFormTitle = (id, detailsUrl, title) => (
+  <dt
+    className="vads-u-padding-top--3 vads-u-margin--0 vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-font-weight--bold"
+    data-e2e-id="result-title"
+  >
+    {detailsUrl ? (
+      <a href={detailsUrl} className="vads-u-text-decoration--none">
+        <dfn>
+          <span className="vads-u-visibility--screen-reader">
+            Visit the landing page for Form number
+          </span>
+          {id}{' '}
+        </dfn>
+        {title}
+        <i
+          className="fas fa-angle-right vads-u-margin-left--0p25"
+          style={{ verticalAlign: 'middle' }}
+        />
```

Comment:
I'm surprised it was necessary to have this `verticalAlign`. Usually the icons line up okay by default. We definitely need this though?

I _think_ we need to add `aria-hidden="true" role="presentation"` to this icon so that screen readers don't worry about it

Reply:
Yeah, if you look at the review instance and toggle the verticalAlign style, it should sit on the bottom of the text instead in the middle of the block element.